RecoLocalCalo/HcalRecAlgos/src/HFStripFilter.cc

```diff
@@ -356,15 +356,15 @@ std::unique_ptr<HFStripFilter> HFStripFilter::parseParameterSet(const edm::Param
 edm::ParameterSetDescription HFStripFilter::fillDescription() {
   edm::ParameterSetDescription desc;
 
-  desc.add<double>("stripThreshold", 40.0);
-  desc.add<double>("maxThreshold", 100.0);
-  desc.add<double>("timeMax", 6.0);
-  desc.add<double>("maxStripTime", 10.0);
-  desc.add<double>("wedgeCut", 0.05);
-  desc.add<int>("seedHitIetaMax", 35);
-  desc.add<int>("gap", 2);
-  desc.add<int>("lstrips", 2);
-  desc.addUntracked<int>("verboseLevel", 0);
+  desc.add<double>("stripThreshold", 40.0);   // threshold to include hits into strips
```

Comment:
I meant using the ```setComment("...")``` method here, see
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Adding_Comments

As such the comments will appear also in the automatically generated ```HFStripFilter_cfi.py``` config